docs/website/docs/reference/tuning.md

```diff
@@ -1,36 +1,47 @@
 ---
+hide:
+  - tags
+tags:
+  - performance
 icon: octicons/meter-16
 status: new
 ---
 
 # Tuning
 
-This page documents support for IREE dispatch tuning. The compiler supports
-both default and user-provided tuning specs (specifications) that override
-compiler heuristics that guide dispatch code generation. In our experience,
-tuning specs can provide meaningful speedup of model execution. For example, we
-achieved a ~10% improvement on the Stable Diffusion XL (SDXL) model with the
-MI300X GPU.
+Tuning in IREE is used to generate transform dialect (TD) specs that guide the
+IREE compiler in selecting optimal values for configurable parameters during
+the compilation phase.
 
-## Tuning specs
+The IREE compiler is shipped with target-specific, default specs. With these
+default specs, we aim to cover the most in-demand hardware and dispatches from
+the most popular ML models, although we do not guarantee completeness. So for
+many users, the default specs will provide sufficient performance, but are far
+from perfect.
 
-The default specs are shipped with the IREE compiler and are target-specific.
-We aim to provide default tuning specs that cover the most in-demand hardware
-and dispatches from most popular ML models, although we do not guarantee
-completeness.
+For users looking to get the best performance on custom models and hardware
+targets, the IREE compiler also supports user-provided tuning specs. These
+specs can be made manually or generated using our dispatch tuner, [SHARK Tuner](https://github.com/nod-ai/shark-ai/tree/main/sharktuner).
 
-User-provided tuning specs are a mechanism that allows for users to get the
-best performance on custom models and hardware targets without having to modify
-the compiler source code or needlessly special-case compiler heuristics.
+A dispatch tuner is simply a tuner which performs tuning on units of 
```

Comment:
trim trailing space to appease the pre-commit check: https://github.com/iree-org/iree/actions/runs/14867173837/job/41746733185?pr=20704#step:4:183
```suggestion
A dispatch tuner is simply a tuner which performs tuning on units of
```